en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands.
+
+By default on first boot, stock Core images run a program called console-conf. If the you have a keyboard and monitor attached to the device or have a serial connection, you can use console-conf to configure the network and add a user account. The user account is created based on your registered email address and ssh key. This user account can then be used to ssh to the device for various management purposes. (Technically, this is not a system user account because it is not created through assertion, but the end result is equivalent.)
+
+Some systems suppress console-conf,  and a user is not created by default. In these cases, one may add a system-user if one has the information and authority needed to do so for the particular system. This is done by inserting a USB drive with a special file (named `auto-import.assert`) in the root directory. Snapd imports the assertions found in it, and, if the assertions are valid for the given system, the system-user is created. After this, you can log into the device (locally or over ssh) using the username and password defined.
+
+In general, you cannot add a system-user to a system that is already "managed". A system is managed if a user was already created by console-conf, or if a system-user was created through the auto-import mechanism described here.
+
+Whether you can add a system-user to an unmanaged system depends on the model assertion that was used to create the image. It also depends on whether you have an authorized snapcraft key on your local system. For a discussion of keys, see [Ubuntu Core Images](../build-device/image-building.md).
```

Comment:
I've modified the incorrect image-building doc to use snapcraft command for keys (instead of snap command, which is now incorrect) and have change the reference to the key from "signature key" to "snapcraft key"